### PR TITLE
add disk validation in delete()

### PR DESCRIPTION
**Pull request by pxpm**
Non breaking change for current version, implemented in the future v4 already. 

Migth need to resolve v4 conflicts after this is merged since this bit was refactored into new function.

### CODE REVIEW

File: src/app/Http/Controllers/BackupController.php [hunk 1]
```diff
@@ -113,7 +113,13 @@ public function download()
      */
     public function delete($file_name)
     {
-        $disk = Storage::disk(Request::input('disk'));
+        $diskName = Request::input('disk');
+
+        if (!in_array($diskName, config('backup.backup.destination.disks'))) {
+            abort(500, trans('backpack::backup.unknown_disk'));
```
Comment (tabacitu) [inline src/app/Http/Controllers/BackupController.php:119]: Does this lang string exist in the old version too?

**Comment by pxpm** [comment]
@tabacitu I saw you somewhere in my email asked about the `unknow_disk` key, I can't find the comment here in github, hey, but my notifications are working!! - it exists since the last tagged version.

**Comment by pxpm** [comment]
@tabacitu I saw you somewhere in my email asked about the `unknow_disk` key, I can't find the comment here in github, hey, but my notifications are working!! - it exists since the last tagged version.

**Review by promatik — APPROVED** [review]
As @pxpm said, I approve this for v3, since #105 (v4) already has this 👌

**Comment by tabacitu** [comment]
Damn it! This was pointing to `master`, which was basically `v4`, so by mistake I merged it there. I've just pushed a separate commit that does the same thing to `v3`.

Until we launch v4, let's please use the `v3` and `v4` branches only (no `master`, otherwise it'll get confusing.

**Pull request merged**